tests/utils/test_utils.py

```diff
@@ -0,0 +1,12 @@
+from telcell.utils.utils import is_colocated
+from telcell.data.parsers import parse_measurements_csv
+
+
+def test_is_colocated(path='./tests/testdata.csv'):
+    tracks = parse_measurements_csv(path)
+    track_a = tracks[0]
+    track_b = tracks[1]
+    track_c = tracks[2]
```

Comment:
```suggestion
    track_a, track_b, track_c = parse_measurements_csv(path)
```

Would it make sense to validate that a track is colocated to itself?

Reply:
IMO it makes sense to assign a colocation label when two tracks are paired.